package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>buoy_msgs</name>
+  <version>0.0.0</version>
+  <description>Power Buoy messages (telemetry) and services (commands)</description>
+  <maintainer email="anderson@mbari.org">Michael Anderson</maintainer>
+  <license>TODO: License declaration</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  
+  <build_depend>rosidl_default_generators</build_depend>
```

Comment:
I believe this is usually declared as `buildtool_depend`. See [for example](https://github.com/ros2/common_interfaces/blob/f3cb4848560e91596e7688e8ac1816828fa460cb/geometry_msgs/package.xml#L19).

```suggestion
  <buildtool_depend>rosidl_default_generators</buildtool_depend>
```